<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event4</class>
 <widget class="QWidget" name="Event4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wondercard IVs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Diamond</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabRNGSelector">
     <widget class="QWidget" name="tabGenerator">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">100000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorDelay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="TextBox" name="textBoxGeneratorDelay"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSpecies">
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxGeneratorSpecies"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxGeneratorLevel">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorNature">
            <property name="text">
             <string>Nature</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxGeneratorNature"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterGenerator" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewGenerator"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherMinDelay">
            <property name="text">
             <string>Min Delay</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxSearcherMinDelay">
            <property name="text">
             <string notr="true">600</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelSearcherMaxDelay">
            <property name="text">
             <string>Max Delay</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="TextBox" name="textBoxSearcherMaxDelay">
            <property name="text">
             <string notr="true">2000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherMinAdvance">
            <property name="text">
             <string>Min Advance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxSearcherMinAdvance">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelSearcherMaxAdvance">
            <property name="text">
             <string>Max Advance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="TextBox" name="textBoxSearcherMaxAdvance">
            <property name="text">
             <string notr="true">1000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxSearcherSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherSpecies">
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxSearcherSpecies"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxSearcherLevel">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherNature">
            <property name="text">
             <string>Nature</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxSearcherNature"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxSearcherFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterSearcher" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBar"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="TableView" name="tableViewSearcher">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabRNGSelector</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>textBoxGeneratorDelay</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxGeneratorSpecies</tabstop>
  <tabstop>spinBoxGeneratorLevel</tabstop>
  <tabstop>comboBoxGeneratorNature</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>textBoxSearcherMinDelay</tabstop>
  <tabstop>textBoxSearcherMaxDelay</tabstop>
  <tabstop>textBoxSearcherMinAdvance</tabstop>
  <tabstop>textBoxSearcherMaxAdvance</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxSearcherSpecies</tabstop>
  <tabstop>spinBoxSearcherLevel</tabstop>
  <tabstop>comboBoxSearcherNature</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
